packages/rolldown/src/api/dev/dev-options.ts

```diff
@@ -1,8 +1,26 @@
 import type { BindingHmrUpdate } from '../../binding';
 
 export interface DevWatchOptions {
+  /**
+   * If `true`, use polling instead of native file system events for watching.
+   * @default false
+   */
   usePolling?: boolean;
+  /**
+   * Poll interval in milliseconds (only used when usePolling is true).
+   * @default 100
+   */
   pollInterval?: number;
+  /**
+   * If `true`, use debounced watcher. If `false`, use non-debounced watcher for immediate responses.
+   * @default true
+   */
+  useDebounce?: boolean;
+  /**
+   * Debounce duration in milliseconds (only used when useDebounce is true).
+   * @default 10
+   */
+  debounceDuration?: number;
```

Comment:
I wonder if we can use a single option for this, disabling debounce if `debounceDuration` is 0.

Reply:
Yes. I thought about it and then decide to follow `usePolling` option pattern(since it already exists.

Reply:
Ah, didn't thought about `debounce: 0` merging the immediate events. That makes sense.